Libraries/plist/Tools/PlistBuddy.cpp

```diff
@@ -241,183 +216,235 @@ ParseCommandValueString(std::vector<std::string>::const_iterator begin, std::vec
 }
 
 static plist::Object *
-GetObjectAtKeyPath(plist::Object *object, std::queue<std::string> *remainingKeys, bool leaveLastKey = true)
+GetObjectAtKeyPath(plist::Object *object, std::queue<std::string> *keyPath, bool leaveLastKey = true)
```

Comment:
This feels to me too much of code just to make recursion call to a iterative one...